vcr/api/v2/api.go

```diff
@@ -96,6 +98,12 @@ func (w Wrapper) IssueVC(ctx echo.Context) error {
 		public = *issueRequest.Visibility == IssueVCRequestVisibilityPublic
 	}
 
+	// Set default context, if not set
+	if issueRequest.Context == nil {
```

Comment:
`https://www.w3.org/2018/credentials/v1` must be added as well

Reply:
The default VC context is added by the issuer (also stated in the OAS) (see `vcr/issuer/issuer.go:109`).

Reply:
🤷‍♂️